### PR TITLE
2.0-preview branch

### PR DESCRIPTION
**Pull request by bicknellr**
The ripple doesn't work right now as this is waiting on an update to paper-behaviors.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,20 +21,30 @@
   "homepage": "https://github.com/PolymerElements/paper-checkbox",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.6.0",
+    "polymer": "Polymer/polymer#2.0-preview",
     "iron-checked-element-behavior": "PolymerElements/iron-checked-element-behavior#^1.0.0",
     "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
```
Comment (bicknellr) [inline bower.json]: This needs to be updated once paper-behaviors is ported.

File: test/index.html [hunk 2]
```diff
@@ -15,12 +15,12 @@
 <body>
   <script>
     WCT.loadSuites([
-      'basic.html',
       'basic.html?dom=shadow',
       // To enable `useNativeCSSProperties`, the native Shadow DOM and lazy
       // registration flags must also be enabled.
       // https://github.com/Polymer/polymer/blob/ff6e884ef4f309d41491333860a8bc9c2f178696/src/lib/settings.html#L55
-      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true'
+      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true',
```
Comment (e111077) [inline test/index.html]: Is there any particular reason as to why we are running tests with these extra flags?

Comment (e111077) [inline test/index.html]: scratch that just read the blame commit message

File: .travis.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 language: node_js
 sudo: required
 before_script:
-  - npm install -g bower polylint web-component-tester eslint eslint-plugin-html
-  - bower install
-  - polylint
+  - npm install -g bower web-component-tester@next eslint eslint-plugin-html polymer-cli@next
```
Comment (e111077) [inline .travis.yml]: Can you please change the travis.yml to look like the ones from other repos?

**Comment by bicknellr** [comment]
Needs `@apply(x);` -> `@apply x;` in a few places in `paper-checkbox.html`.

**Comment by bicknellr** [comment]
fixed

**Pull request merged**